pkg/apis/pingcap/v1alpha1/backup.go

```diff
@@ -321,13 +337,21 @@ func ParseLogBackupSubcommand(backup *Backup) LogSubCommandType {
 	if backup.Spec.Mode != BackupModeLog {
 		return ""
 	}
-	if backup.Spec.LogStop {
+	switch backup.Spec.LogSubcommand {
+	case "log-start":
+		if IsLogBackupAlreadyPaused(backup) {
+			return LogResumeCommand
+		}
+		return LogStartCommand
+	case "log-stop":
 		return LogStopCommand
-	}
-	if backup.Spec.LogTruncateUntil != "" {
```

Comment:
Is it needed to compatible with the old fields?

Reply:
No, user shouldn't mix up these two interface to avoid mistakes

Reply:
So the old existing Backup user MUST update their pipeline to use these new fields, right?

Reply:
Yes, we don't want handle corner case like:
````
Spec:
    logSubcommand: log-start
    logStop: true
```
BTW, I am fresh to operator. Would there be any problem?
I tested updating operator online while a pitr task is running.
Seems there wouldn't be any problem.

Reply:
We need to keep compatible with the old fields, because tidbcloud depends on current implementation. I think we can make old fields have higher priority. For example:
```
if backup.Spec.LogStop
    return LogStopCommand
if backup.Spec.LogTruncateUntil != "" && backup.Spec.LogTruncateUntil != backup.Status.LogSuccessTruncateUntil
    return LogTruncateCommand

switch backup.Spec.LogSubcommand
...

// default command is log start
return LogStartCommand
```

Reply:
Seems `backup.Spec.LogTruncateUntil != backup.Status.LogSuccessTruncateUntil` is unnecessary
The user should be allowed to truncate multiple times.